src/components/Channel/channelState.ts

```diff
@@ -111,7 +234,6 @@ export const channelReducer = (state, action) => {
   }
 };
 
-/** @type {import('./types').ChannelState} */
 export const initialState = {
```

Comment:
should this have `initialState: ChannelState`?

Reply:
since ChannelState needs generics throwing this on there led to typing errors